elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/metadata/SQLPhysicalTable.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.queryengines.sql.metadata;
+
+import static com.yahoo.elide.datastores.aggregation.queryengines.sql.metadata.SQLTable.resolveSQLDimensions;
+
+import com.yahoo.elide.datastores.aggregation.AggregationDictionary;
+import com.yahoo.elide.datastores.aggregation.metadata.models.Table;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.Set;
+
+/**
+ * SQL extension of {@link Table} which also contains sql column meta data.
+ */
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class SQLPhysicalTable extends Table implements SQLTable {
```

Comment:
This reads like a physical table is a logical table.  If we need two concepts, we should create LogicalTable and PhysicalTable and there should be an inheritance linkage between them.

Reply:
👍